Library/Homebrew/extend/hash_validator.rb

```diff
@@ -1,13 +0,0 @@
-# typed: false
-# frozen_string_literal: true
-
-module HashValidator
-  refine Hash do
-    def assert_valid_keys!(*valid_keys)
```

Comment:
Note that the `ActiveSupport` implementation [varies](https://github.com/rails/rails/blob/v6.1.7.3/activesupport/lib/active_support/core_ext/hash/keys.rb#L39-L55) a bit, although this shouldn't matter, based on my cursory look at our actual call sites.